app/build.gradle

```diff
@@ -20,7 +20,7 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled true
+            minifyEnabled false
```

Comment:
we should have minifyEnabled in release versions to have the generated apk optimized as much as possible, also please note that keeping the minifyEnabled to true might impact the building of some project deps, so please after generating the apk give it a try to verify it doesn't crash